Sources/Logging/Docs.docc/ImplementingALogHandler.md

```diff
@@ -0,0 +1,171 @@
+# Implementing a log handler
+
+A guide for implementing custom logging backends that conform to the
+``LogHandler`` protocol.
```

Comment:
```suggestion
Create a custom logging backend that provides logging services for your apps and libraries.
```

You generally don't want to use code voice or links to symbols in abstracts within DocC, as they don't reflect the links correctly, since they're rendered in multiple areas. It's better to spell it out narratively for that first sentence aligned with the title.

I also rephrased the abstract in the suggestion to describe what this article covers (how to conform, etc) and why you'd do it, not mentioning the specific protocol in the abstract.